color.d

```diff
@@ -1777,14 +1779,45 @@ struct Point {
 	Point opBinary(string op)(int rhs) @nogc {
 		return Point(mixin("x" ~ op ~ "rhs"), mixin("y" ~ op ~ "rhs"));
 	}
+
+	Size opCast(T = Size)() inout @nogc {
+		return Size(x, y);
+	}
 }
 
 ///
 struct Size {
 	int width; ///
 	int height; ///
 
-	int area() pure nothrow @safe const @nogc { return width * height; }
+	pure nothrow @safe:
+
+	int area() const @nogc { return width * height; }
+
+	int offset(const int x, const int y) const @nogc {
```

Comment:
hmm how do you use this? i wonder if it would be a better fit somewhere else (even just UFCSing the linearOffset function) cuz a Size doesn't really have an offset so im just not sure it makes sense in isolation here (heck even as i read this i thought it was in `Rectangle` at first then saww it was in Size)

Reply:
Let’s revert this addition then

Reply:
ok, is that gonna be a hassle for your use case? cuz whatever we put in here (ever since i tag 11.4,  it will be subject to random breakage until then as a beta period) we'll be committing to so just wanna make sure we have what is useful to you even as i take the axe to the pr lol

Reply:
no, it’s fine :P

Reply:
I’ve already removed it from this patch.